src/ker_lowupsampfac_horner_allw_loop_constexpr.h

```diff
@@ -0,0 +1,201 @@
+// Header of static arrays of monomial coeffs of spreading kernel function in each
+// fine-grid interval. Generated by gen_all_horner_cpp_header.m in finufft/devel
+// Authors: Alex Barnett, Ludvig af Klinteberg, Marco Barbone & Libin Lu.
+// (C) 2018--2024 The Simons Foundation, Inc.
+#include <array>
+
+template<uint8_t w> constexpr auto nc125() noexcept {
+  constexpr uint8_t ncs[] = {4, 5, 6, 6, 7, 8, 8, 9, 10, 10, 11, 12, 13, 13, 14, };
```

Comment:
this should be `constexpr std::array<uint8_t, 14> ncs{4, 5, 6, 6, 7, 8, 8, 9, 10, 10, 11, 12, 13, 13, 14,}`